pkg/api/types.go

```diff
@@ -2051,7 +2051,7 @@ type SecurityContext struct {
 	// RunAsNonRoot indicates that the container should be run as a non-root user.  If the RunAsUser
 	// field is not explicitly set then the kubelet may check the image for a specified user or
 	// perform defaulting to specify a user.
-	RunAsNonRoot bool
+	RunAsNonRoot bool `json:"runAsNonRoot,omitempty"`
```

Comment:
@erictune I think I actually want to hold off on this for now until the volume security stuff is in.  There are some questions about pointers in this API from folks that I haven't had time to fully sort out with them yet, so perhaps I should tag this 'hold' for now.